infra/notifications.js

```diff
@@ -0,0 +1,124 @@
+const Notification = require('../models/notification');
+const templates = require('../constants/notifications');
+
+/**
+ *
+ * @param {String} fromUser user who answered the post
+ * @param {String} toUser user whose post was answered
+ * @param {String} forumId id of the corresponding post
+ */
+const createForumNotification = async (fromUser, toUser, forumId) => {
+  let body = {
+    fromUser,
+    toUser,
+    title: 'Forum',
+    message: templates.forumTemplate({ user: fromUser, question: forumId }),
+    eventId: forumId,
+    model: 'Forum',
```

Comment:
Since this value is used multiple times, we can separate this title to constants too.
constants.MODEL_FORUM
So in case we can't to change the title